src/utils/aad/aad.account-management.constants.ts

```diff
@@ -0,0 +1,4 @@
+export const AAD_ACCOUNT_MANAGEMENT_MODULE_OPTIONS = 'aadModuleOptions';
+export const AAD_ACCOUNT_MANAGEMENT_MODULE_PROVIDER = 'aadModule';
```

Comment:
All these constants are used within one file so probably best to have them in that file as not used elsewhere? Also names (i.e. values) do need an update?

Reply:
Completely disagree - that's standard notation and it's better to have 10 files with 5 lines each with clear distinction than 1 file with 10 purposes and 50 lines. Names are tokens and can be anything - agreed it will be cleaner to change the strings but to be honest a guid will do the same (and quite frankly might be better, as this tokens need to be unique, otherwise injecting them will fail).

Reply:
https://app.zenhub.com/workspaces/cherrytwist-5ecb98b262ebd9f4aec4194c/issues/cherrytwist/server/765

Reply:
Improved / fixed in ccce2a9f05949ea416259c7af1ca183ba955375f